mythril/laser/ethereum/state/environment.py

```diff
@@ -41,7 +41,10 @@ def __init__(
         self.active_function_name = ""
 
         self.address = active_account.address
+
+        # TODO: Block number should increase with transactions
         self.block_number = symbol_factory.BitVecSym("block_number", 256)
+        self.chainid = symbol_factory.BitVecSym("chain_id", 256)
```

Comment:
Maybe we should use a default concrete value here? (cli option maybe)
Different chain_id's could in theory have different opcode implementations right?

Reply:
(note to self: tox.ini is also changed) 😄

Reply:
Yes, but we can leave it at the moment, as users won't get anything out of it now.